### ISSUE TITLE
what's the tpr when far is at 0.0001%?

### ISSUE THREAD

**Issue by hustzeyu** [zhanglaplace/MobileFaceNet#3]
what's the tpr when far is at 0.0001%?

**Comment by zhanglaplace** [comment]
@hustzeyu 这个没有测，自己只是随便跑的一个。抱歉

**Comment by piaopiaoxiaoye** [comment]
Can I use this caffemodel on Android platform or linux platform? Is this your already trained "mobilefacenet" model?
Thank you!
best regards!

**Comment by shiyuanyin** [comment]
> 
> 
> @hustzeyu 这个没有测，自己只是随便跑的一个。抱歉

作者您好，
我用的vggface2,下载的训练集8631，的数据进行训练，损失函数和您的模型也是一样的，训练的时候从12左右的损失值开始减少，但是，并没有收敛，损失值在10左右震荡，不继续减小，作者您遇到过这个情况吗？？？可能有哪些原因昵

**Comment by zhanglaplace** [comment]
@piaopiaoxiaoye 
Andrioid : using [ncnn](https://github.com/Tencent/ncnn) and convert caffe model to ncnn model
linux : Just compile caffe

**Comment by zhanglaplace** [comment]
@shiyuanyin learning_rate and batch_size ?

**Comment by shiyuanyin** [comment]
> 
> 
> @zhanglaplace 
lr 0.1 开始， batchsize=64

**Comment by shiyuanyin** [comment]
> 
> 
> @shiyuanyin learning_rate and batch_size 
需要，预训练sortmax 的model吗？我直接训练，除了数据不是9136（我的8136），不能收敛，一直震荡

> 
> 
> @zhanglaplace 
> Andrioid : using [ncnn](https://github.com/Tencent/ncnn) and convert caffe model to ncnn model
> linux : Just compile caffe

需要，预训练sortmax 的model吗？我直接训练，除了数据不是9136（我的8136），不能收敛，一直震荡

**Comment by zhanglaplace** [comment]
@shiyuanyin Try to increase the learning rate.  In addition, make sure your training data and labels are aligned。

**Comment by borisgribkov** [comment]
@shiyuanyin Did you solve the problem? I see the same behavior with ResNet-18 with ArcLoss and VGG2 training set, I guess that alignment is important

**Comment by borisgribkov** [comment]
@shiyuanyin I guess I fixed the problem, in case of not aligned vgg2 dataset training is very sensitive to M parameter, try to change
